src/main/resources/migration/v3_0_0_2.js

```diff
@@ -0,0 +1,10 @@
+db.serverSettings.update(
+    {},
+    {$rename: {"serverEmailConfig":"serverEmailDetails"}})
+db.serverSettings.update(
+    {},
+    {
+        $set: {"_class": "com.epam.ta.reportportal.database.entity.settings.ServerSettings",
+            "serverEmailDetails._class": "com.epam.ta.reportportal.database.entity.settings.ServerEmailDetails"}
+    }
+)
```

Comment:
Please, split to two scripts. Make sure field is present before updating it:
db.serverSettings.update(
    {'_class': {$exists : false}}},
    {
        $set: {"_class": "com.epam.ta.reportportal.database.entity.settings.ServerSettings"}
    }
)